app/src/components/Shared/MenuMobile.jsx

```diff
@@ -11,28 +11,31 @@ class MenuMobile extends Component {
     const backClass = this.props.visible ?
       `${menuMobile['menu-back']} ${menuMobile['-show']}` : `${menuMobile['menu-back']}`;
 
-    let userLinks;
-    if (this.props.loggedUser) {
-      userLinks = (
-        <li>
-          <a>{this.props.loggedUser.displayName}</a>
-          <ul>
-            <li>
-              <a onClick={this.logout}>
-                Logout
-              </a>
-            </li>
-          </ul>
-        </li>
-      );
-    } else {
-      userLinks = (
-        <div>
-          <span className={menuMobile['button-login']} onClick={this.props.login}>LOG IN</span>
-          <p className={menuMobile['forgot-text']}>Forgot your password? Not a member</p>
-        </div>
-      );
-    }
+    // there's no design when user is logged so I commented this part
```

Comment:
the user cannot see the map until he is logged in, so please talk with dani ASAP